src/main/kotlin/no/nav/syfo/jobbforventning/infrastructure/kafka/Oppfolgingstilfelle.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.syfo.jobbforventning.infrastructure.kafka
+
+import no.nav.syfo.shared.domain.Personident
+import no.nav.syfo.shared.util.*
+import java.time.*
+import java.time.temporal.ChronoUnit
+import java.util.*
+
+// TODO: Move to Jobbforventning domain class
+private const val JOBBFORVENTNING_STOPPUNKT_START_WEEKS = 6L
+private const val JOBBFORVENTETNING_STOPPUNKT_INTERVAL_DAYS = 30L
```

Comment:
Tar utgangspunkt i uke 6 og med 30 dagers intervall. Vi kommer nok til å spikre dette mer etter sommeren, men bruker dette som et utgangspunkt for å teste ting i dev nå.